include/ck_tile/ops/gemm/kernel/universal_gemm_kernel.hpp

```diff
@@ -708,7 +708,7 @@ struct UniversalGemmKernel
                     make_tuple(kargs.M, kargs.N),
```

Comment:
When layout is ColumnMajor then this should be 
```suggestion
                    make_tuple(kargs.N, kargs.M),
```